Tools/CISupport/ews-build/steps.py

```diff
@@ -5502,7 +5519,10 @@ def getResultSummary(self):
         return super().getResultSummary()
 
     def doStepIf(self, step):
-        return self.getProperty('reviewers_full_names')
+        classification = self.getProperty('classification', [])
+        if not isinstance(classification, list):
+            classification = []
+        return self.getProperty('reviewers_full_names') and ['Cherry-pick'] != classification
```

Comment:
should it be `'Cherry-pick' not in classification` or you need exact match of both lists?